Android/app/src/main/res/layout/activity_profile.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorPrimary"
+    tools:context=".ProfileActivity">
+
+    <TextView
+        android:id="@+id/display_exp"
+        android:layout_width="wrap_content"
+        android:layout_height="46dp"
+        android:layout_above="@+id/change_image_btn"
+        android:layout_centerHorizontal="true"
+        android:paddingTop="6sp"
+        android:text="Member Since"
+        android:textColor="#333434"
+        android:textSize="18sp" />
+
+    <TextView
+        android:id="@+id/display_email2"
+        android:layout_width="254dp"
+        android:layout_height="45dp"
+        android:layout_alignParentBottom="true"
+        android:layout_alignStart="@+id/display_exp"
+        android:layout_marginBottom="136dp"
+        android:paddingTop="6sp"
+        android:text="Display Email"
+        android:textColor="#333434"
+        android:textSize="18sp" />
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/settings_image"
+        android:layout_width="220dp"
+        android:layout_height="220dp"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="24dp"
+        android:src="@drawable/default_user_icon" />
+
+    <Button
+        android:id="@+id/change_image_btn"
```

Comment:
what is this button for?

Reply:
To change profile pic. Shall I remove it?